packages/client/components/TopBarNotifications.tsx

```diff
@@ -43,6 +48,9 @@ const TopBarNotifications = ({queryRef}: Props) => {
   const {viewer} = data
   const notifications = viewer?.notifications || {edges: []}
   const {edges} = notifications
+  const snackbarNotifications = edges.filter(
```

Comment:
+2 I like this idea. It'll be easy to add further snackbar notifications in the future if we want to